modules/get-started/pages/whats-new.adoc

```diff
@@ -21,6 +21,13 @@ You can now deploy data transform functions that xref:develop:data-transforms/de
 
 The docs now also include an xref:develop:data-transforms/index.adoc[expanded guide] designed to help you master the creation, deployment, and management of data transforms in Redpanda.
 
+
+== Client throughput management
+
+Redpanda now includes `rpk` and Redpanda Console support for managing xref:manage:cluster-maintenance/manage-throughput.adoc#client-throughput-limits[client throughput limits]. You can set client throughput quotas using `rpk` for individual client based on client ID, as well as a group of clients based on ID prefix matching. New xref:manage:cluster-maintenance/manage-throughput.adoc#monitor-client-throughput[metrics] are available for insight on client throughput utilization.
```

Comment:
@mattschumpert I removed the specific mention of AlterClientQuotas and DescribeClientQuotas. Is this less confusing? With that said, the advice that I got from @pgellert is to try and push users to `rpk` for simplicity. In terms of using the APIs directly, a tool like `kafka-configs.sh` doesn't support client-id-prefix quotas since those are specific to Redpanda. Console doesn't yet support editing quotas in the UI as far as I'm aware -- maybe @weeco can confirm again.

Reply:
Correct, no editing available - just viewing. We are considering working on this as a stretch but I'm almost confident that we won't be able to include this in the next release, so not relevant for this PR.